### ISSUE TITLE
aria/aria-describedby_attribute Multiple references support

### ISSUE THREAD

**Issue by JeffersonBledsoe** [accessibilitysupported/a11ysupport.io#272]
Similarly to how we have a [two ref test for `aria-labeledby`](https://a11ysupport.io/tests/tech__aria__aria-labelledby-two-ref), [aria-describedby](https://www.w3.org/TR/wai-aria-1.1/#aria-describedby) has support for multiple references too.

**Comment by mfairchild365** [comment]
Yes, I agree this would be good to add. Thank you for raising this.
